meta-k8s-setup/lib/oeqa/runtime/cases/config.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+import os
+from oeqa.runtime.case import OERuntimeTestCase
+from oeqa.utils.decorators import *
+from os import listdir
+from os.path import isfile, join
+    
+class ConfigPartitionTest(OERuntimeTestCase):
+
+    def test_config_present(self):
+        # Check to see that the config tar is correctly mounted
+        (status, output) = self.target.run(f"ls /config | grep config.tar.gz -c")
+        self.assertEqual(status, 0, msg=output)
```

Comment:
Shelling out for this is a bit weird. Try something like this:
```suggestion
        self.assertTrue(os.path.exists("/config/config.tar.gz"))
```

Reply:
Well the command does not work when using it in the tests.

Reply:
Hmm, make sense, a bit limiting tho.